src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -46,8 +55,26 @@
     private static final String HEADER_CONTENT_TYPE = "Content-Type";
     private static final String CONTENT_TYPE_FORM_URLENCODED = "application/x-www-form-urlencoded; charset=utf-8";
     private static final Pattern CHARSET_PATTERN = Pattern.compile("(?i)\\bcharset=\\s*\"?([^\\s;\"]*)");
+    private static final X509TrustManager TRUST_SELF_SIGNED_CERTS = new X509TrustManager() {
+        @Override
+        public void checkClientTrusted(X509Certificate[] chain, String authType) throws CertificateException {
+            throw new CertificateException("This trust manager cannot be used in a server");
+        }
+
+        @Override
+        public void checkServerTrusted(X509Certificate[] chain, String authType) throws CertificateException {
+            if (chain.length != 1) {
```

Comment:
guter Ansatz, bei self signed Certs hast ja keine CA die in der Kette drüber hängt 👍